setup.py

```diff
@@ -179,10 +182,17 @@ def run(self):
     author=__author__,
     author_email='info@ansible.com',
     url='https://ansible.com/',
+    project_urls={
+        'Bug Tracker': 'https://github.com/ansible/ansible/issues',
+        'CI: AppVeyor': 'https://app.shippable.com/github/ansible/ansible',
```

Comment:
We use Shippable, not AppVeyor, so I assume this should be `CI: Shippable` instead?

Reply:
Are there any rules about what keys should be?